addons/api/addon/services/sqlite.js

```diff
@@ -118,7 +118,7 @@ export const modelMapping = {
     id: 'id',
     name: 'name',
     description: 'description',
-    scope_id: 'scope_id',
+    scope_id: 'scope.scope_id',
```

Comment:
Fix issue mentioned in this [comment](https://github.com/hashicorp/boundary-ui/pull/3056/files#r2516212059)
cc: @ZedLi